pkg/middleware/is_duplicated_middleware.go

```diff
@@ -0,0 +1,40 @@
+package middleware
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	restError "github.com/kotalco/community-api/pkg/errors"
+	"github.com/kotalco/community-api/pkg/k8s/statefulset"
+	"k8s.io/apimachinery/pkg/types"
+	"net/http"
+)
+
+var statefulService = statefulset.NewService()
+
+func IsDuplicated(c *fiber.Ctx) error {
+	if c.Request().Header.IsPost() { // check on posts verbs only while trying to create new resource
```

Comment:
remove the `IsPost` check, because we've explicitly used the middleware in .Create handlers with Post method.